reedsolomon/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/tron-us/go-btfs/reedsolomon
+
+go 1.12
+
+require (
+	github.com/klauspost/cpuid v1.2.1 // indirect
+	github.com/klauspost/reedsolomon v1.9.2
```

Comment:
Should use the central go.mod, not it's own module since go-btfs itself is not a library

but this package, to go-btfs, is a "module"